Core/Inc/dti.h

```diff
@@ -61,14 +73,19 @@ dti_t *dti_init();
  */
 void dti_set_torque(int16_t torque);
 
-// TODO: Regen interface
-// void dti_set_regen(int16_t regen);
+/**
+ * @brief Set the brake AC current target for regenerative braking. Only positive values are accepted by the DTI.
+ * 
+ * @param current_target The desired AC current to do regenerative braking at. Must be positive.
+ */
+void dti_set_regen(int16_t current_target);
 
-/*
- * SCALE: 10
- * UNITS: Amps
+/**
+ * @brief Send a CAN message containing the AC current target for regenerative braking.
+ * 
+ * @param brake_current AC current target for regenerative braking. The motor controler has a maximum resolution of the tenths place.
  */
-void dti_set_brake_current(int16_t brake_current);
+void dti_send_brake_current(float brake_current);
```

Comment:
dont love working with floats - why cant we have the times factor of 10 to maintain precision earlier than this. Id rather eliminate floats at the earliest stage of data processing rather than carry it out for an arbitrary length. Means fewer floats + consistint rule that we can trust will always be the case

Reply:
I'm good with this. I'll change the interfaces to take a uint16 * 10.